react/finger-exercises/ecmascript/02 - promises/test.js

```diff
@@ -3,18 +3,19 @@ import { delay, asyncDelay } from '.';
 const TIME = 500;
 
 describe('Promises', () => {
-  xit('delay works well', () => delay(TIME)
+  it('delay works well', () => delay(TIME)
     .then(delayedTime => {
       expect(delayedTime).toBeGreaterThanOrEqual(TIME - 100);
       expect(delayedTime).toBeLessThanOrEqual(TIME + 100);
```

Comment:
Are you sure that all the cases passes?, because i think you are not including this case and the previus one.